src/main/kotlin/no/nav/familie/ef/sak/oppgave/TilordnetRessursService.kt

```diff
@@ -35,12 +38,11 @@ class TilordnetRessursService(
         }
     }
 
-    fun hentIkkeFerdigstiltOppgaveForBehandling(behandlingId: UUID, oppgavetyper: Set<Oppgavetype>? = null): Oppgave? {
-        val typer = if (oppgavetyper.isNullOrEmpty()) setOf(Oppgavetype.BehandleSak, Oppgavetype.BehandleUnderkjentVedtak) else oppgavetyper
-
-        return hentEFOppgaveSomIkkeErFerdigstilt(behandlingId, typer)
-            ?.let { oppgaveClient.finnOppgaveMedId(it.gsakOppgaveId) }
-    }
+    fun hentIkkeFerdigstiltOppgaveForBehandling(
+        behandlingId: UUID,
+        oppgavetyper: Set<Oppgavetype> = setOf(Oppgavetype.BehandleSak, Oppgavetype.BehandleUnderkjentVedtak),
```

Comment:
Dette ser bedre ut, men trenger denne egentlig ha default verdi på oppgavetyper (har ikke sjekket bruk)

Reply:
Yep! De andre stedene i løsningen brukes denne metoden for å sjekke om man kan redigere behandlingen og da skal ikke oppgavetyper for godkjenneVedtak tas med i betraktningen. Men akkurat når det kommer til å underkjenne/godkjenne vedtak må man altså gjøre en sjekk på dette